addon/initializers/browser/leaflet-assets.js

```diff
@@ -1,7 +1,7 @@
 /* global L */
 
 export function initialize(/* container, application */) {
-  L.Icon.Default.imagePath = '/assets/images/';
+  //L.Icon.Default.imagePath = '/assets/images/';
```

Comment:
Yes - I'm just not clear on where to add it. 

The issue I ran into with this may be unique to loading ember-leaflet into an engine, and then loading that engine into an Ember App. 
As you can see, I moved this initializer into `/browser` and combined with `fastboot-filter-initializers`, it *should* not be executed in node.

This was the case when I added ember-leaflet into the fastboot-ed Ember app, but once I added it to the Engine, I started to get errors on this line from Node.